src/features/user/RegisterTrees/RegisterTreesForm.tsx

```diff
@@ -49,15 +49,7 @@ const dialogSx: SxProps = {
   },
 };
 
-type FormData = {
-  treeCount: string;
-  species: string;
-  plantProject: string | null;
-  plantDate: Date;
-  geometry: any;
-};
```

Comment:
was there any reason that you removed type  formData ?

Reply:
It must've been removed while merging. I will add it again.